lang/LangSource/SC_LanguageConfig.cpp

```diff
@@ -38,12 +38,12 @@ bool SC_LanguageConfig::gPostInlineWarnings = false;
 
 SC_LanguageConfig* gLanguageConfig;
 
-static const char* INCLUDE_PATHS = "includePaths";
-static const char* EXCLUDE_PATHS = "excludePaths";
-static const char* POST_INLINE_WARNINGS = "postInlineWarnings";
-static const char* CLASS_LIB_DIR_NAME = "SCClassLibrary";
-const char* SCLANG_YAML_CONFIG_FILENAME = "sclang_conf.yaml";
-static const char* EXCLUDE_DEFAULT_PATHS = "excludeDefaultPaths";
+static const std::string INCLUDE_PATHS = "includePaths";
+static const std::string EXCLUDE_PATHS = "excludePaths";
+static const std::string POST_INLINE_WARNINGS = "postInlineWarnings";
+static const std::string CLASS_LIB_DIR_NAME = "SCClassLibrary";
+const std::string SCLANG_YAML_CONFIG_FILENAME = "sclang_conf.yaml";
```

Comment:
missed a `static`.

Actually, you can just put the whole block in an anonymous namespace instead.

Reply:
Ah, I see!